public/js/extraFunctions.js

```diff
@@ -875,22 +885,26 @@ let initialize_quarter_dropdown = function(includedTerms) {
       c.innerHTML = selElmnt.options[j].innerHTML;
       c.id = termConverter[j - 1] || "cum";
     }
-    const term = termConverter[parseInt(c.innerHTML[1]) || 0];
+    const term = termConverter[parseInt(c.id[1]) || 0];
     const isAccessibleObj = isAccessible(term, includedTerms);
     $(c)
-      .removeClass(["inaccessible", "hastooltip"])
-      .removeAttr("aria-label")
+      .removeClass(["inaccessible", "tooltip"])
+      .remove(".tooltiptext")
```

Comment:
This seems to be a vestige of the `.tooltiptext` from before the refactoring, meaning that this code presumably no longer properly does its job.